.github/workflows/android-build-and-release.yml

```
@@ -0,0 +1,128 @@
+
+on:
+  push:
+    branches:
+      - main
+      - development
+      - release/*
+      - fast-track/*
+    tags: ["*"]
+  pull_request:
+    types: [ready_for_review, opened, reopened, auto_merge_enabled]
+  workflow_dispatch:
+    inputs:
+      clo:
+        description: "Assets/StreamingAssets/clo.json"
+        required: false
+        type: string
+        default: ""
+
+env:
+  UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
+  UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
+  UNITY_SERIAL: ${{ secrets.UNITY_SERIAL }}
+
+jobs:
+  build:
+    name: Build for ${{ matrix.targetPlatform }}
+    runs-on:
+      labels: ["linux", "8cores"]
+    defaults:
+        run:
+          shell: bash
+    strategy:
+      matrix:
+        projectPath:
+          - nekoyume
+        targetPlatform:
+          - Android
+
+    steps:
+      - name: Set CRLF
+        run: |
+          git config --global core.autocrlf true
+
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+          submodules: true
+
+      - uses: actions/cache@v3
+        with:
+          path: ${{ matrix.projectPath }}/Library
+          key: Library-${{ matrix.projectPath }}-ubuntu-${{ matrix.targetPlatform }}
+          restore-keys: |
+            Library-${{ matrix.projectPath }}-ubuntu-
+            Library-
+
+      - name: Copy clo.json
+        run: |
+          if [ -n "${{ github.event.inputs.clo }}" ]; then
+            echo ${{ github.event.inputs.clo }} > nekoyume/Assets/StreamingAssets/clo.json
+          fi
+
+      - uses: game-ci/unity-builder@v2
+        with:
+          androidKeystoreName: 9c-aos-signing-keystore.keystore # This file won't exist, but this property needs to exist.
+          androidKeystoreBase64: ${{ secrets.ANDROID_KEYSTORE_BASE64 }}
+          androidKeystorePass: ${{ secrets.ANDROID_KEYSTORE_PASS }}
+          androidKeyaliasName: ${{ secrets.ANDROID_KEY_ALIAS_NAME }}
+          androidKeyaliasPass: ${{ secrets.ANDROID_KEY_ALIAS_PASS }}
+          androidAppBundle: true
+          projectPath: ${{ matrix.projectPath }}
+          targetPlatform: ${{ matrix.targetPlatform }}
+          buildMethod: 'Editor.Builder.Build${{ matrix.targetPlatform }}'
+          buildName: ${{ matrix.targetPlatform }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11.0
+
+      - name: Install bundletool
+        run: |
+          tr -d "\r" < .github/bin/install-bundletool.sh > .github/bin/install-bundletool-trimed.sh
+          bash .github/bin/install-bundletool-trimed.sh /tmp
+
+      - name: Setup Keystore and Passwords
+        run: |
+          echo "${{ secrets.ANDROID_KEYSTORE_BASE64 }}" | base64 -d > /tmp/9c-signing-keystore.keystore
+          echo "${{ secrets.ANDROID_KEYSTORE_PASS }}" > /tmp/keystore.pwd
+          echo "${{ secrets.ANDROID_KEY_ALIAS_NAME }}" > /tmp/keyalias.name
+          echo "${{ secrets.ANDROID_KEY_ALIAS_PASS }}" > /tmp/key.pwd
```

Could you rename it like below?

```suggestion
          echo "${{ secrets.ANDROID_KEY_ALIAS_PASS }}" > /tmp/keyalias.pwd
```

---

Good. I'll apply this